src/main/content/_drafts/2018-09-12-build-and-push-spring-boot-docker-images.adoc

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "The boost-maven-plugin: An easy way to build layered Docker images of Spring Boot apps"
+date:   2018-09-12 10:20:00 -0000
+categories: blog
+seo-title: The boost-maven-plugin: An easy way to build layered Docker images of Spring Boot apps - OpenLiberty.io
+seo-description: A simple and efficient way of building and pushing layered Docker images of Spring Boot applications using Boost Maven Plugin.
+blog_description: "A simple and efficient way of building and pushing layered Docker images of Spring Boot applications using Boost Maven Plugin".
+---
+
+
+= The boost-maven-plugin: An easy way to build layered Docker images of Spring Boot apps
+Anjum Fatima <https://github.com/anjumfatima90>
+
+Do you find writing Dockerfiles difficult, but want an efficient docker image of your
+Spring Boot application? Do you want a maven plugin to do your job?
+If the answer is yes, you are at right place.
+
+Writing a Dockerfile to create an efficient Docker image can be a challenging task.
+It requires, understanding techniques to optimize Dockerfiles. Boost Maven plugin
+provides a functionality to build and push efficient Docker images of Spring Boot
+applications using Liberty.
+
+In the blog post link:blog/2018/07/02/creating-dual-layer-docker-images-for-spring-boot-apps.html[Creating Dual Layer Docker images for Spring Boot apps],
+the steps to create an efficient docker image was shown, but to improve the user experience
+we built a maven plugin that can easily be configured into your Spring Boot application, to
+build and push efficient Docker images without the hassle of writing Dockerfiles.
+
+**Note** : The boost-maven-plugin is in its beta stage. 
+
+== Steps to configure Boost Maven plugin
+
+You can configure the `boost-maven-plugin` in the `plugins` section of the pom.xml
+of your Spring Boot application as shown below:
+----
+<plugin>
+  <groupId>io.openliberty.boost</groupId>
+  <artifactId>boost-maven-plugin</artifactId>
+  <version>0.1</version>
+</plugin>
+----
+
+It requires minimal knowledge of Docker including installing and running Docker on your machine.
+
+NOTE: `spring-boot-maven-plugin` must be configured before this plugin in order to create the required executable archive
+of the application.
+
+
+=== Build Docker image
+
+To build a docker image you can add execution goal to the plugin inside pom.xml like:
+----
+<plugin>
+  <groupId>io.openliberty.boost</groupId>
+  <artifactId>boost-maven-plugin</artifactId>
+  <version>0.1</version>
+  <executions>
+    <execution>
+      <goals>
+        <goal>docker-build</goal>
+      </goals>
+    </execution>
+  </executions>
+</plugin>
+----
+`mvn clean package` will create a docker image with a default repository name as `${project.artifactId}`
+ and a default tag `latest`.
+
+Alternatively, you can configure the plugin with no execution goals and do the following steps:
+
+`mvn clean package`
+
+`mvn boost:docker-build`
```

Comment:
Verify with Mike or Charles, but I think you can combine these commands:

mvn clean package boost:docker-build